src/main/kotlin/org/openrewrite/RecipeMarkdownWriter.kt

```diff
@@ -651,17 +651,13 @@ import TabItem from '@theme/TabItem';
     private fun BufferedWriter.writeContributors(recipeDescriptor: RecipeDescriptor) {
         if (recipeDescriptor.contributors.isNotEmpty()) {
             newLine()
-            writeln("## Contributors")
-            writeln(
-                recipeDescriptor.contributors.stream()
-                    .map { contributor: Contributor ->
-                        if (contributor.email.contains("noreply")) {
-                            contributor.name
-                        } else {
-                            "[" + contributor.name + "](mailto:" + contributor.email + ")"
-                        }
-                    }.collect(joining(", "))
-            )
+            writeln("## Contributors\n")
+            // Extract unique contributor names (removing duplicates)
+            val uniqueNames = recipeDescriptor.contributors
+                .map { contributor -> contributor.name }
```

Comment:
We might want to remove accents here as well:
https://www.baeldung.com/java-remove-accents-from-text

Reply:
Perhaps it would be better to already remove these accents in openrewrite/rewrite, just to avoid duplicating the logic in the platform.

Reply:
Now normalizing author names in https://github.com/openrewrite/rewrite-build-gradle-plugin/commit/9343d65212ed0b6aefe841210b8913b0a6ccc7d1
We'll see if we need to make adjustments here, but I'd not expect so.